vendor/engines/saml_authentication/spec/fixtures/idp_metadata.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0"?>
+<EntityDescriptor xmlns="urn:oasis:names:tc:SAML:2.0:metadata" entityID="https://app.onelogin.com/saml/metadata/747214">
+  <IDPSSODescriptor xmlns:ds="http://www.w3.org/2000/09/xmldsig#" protocolSupportEnumeration="urn:oasis:names:tc:SAML:2.0:protocol">
+    <KeyDescriptor use="signing">
+      <ds:KeyInfo xmlns:ds="http://www.w3.org/2000/09/xmldsig#">
+        <ds:X509Data>
+          <ds:X509Certificate>MIIEGjCCAwKgAwIBAgIURh021zf42BsC+rnlP8afhzeISDwwDQYJKoZIhvcNAQEF
```

Comment:
This is okay to have in version control?

Reply:
Yes, it's a public key.